s3same/__init__.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+import travispy
+import yaml
+
+from .travis import travis_encrypt as _travis_encrypt
```

Comment:
You can probably just import this without renaming it now that there's no naming conflict.